tests/test_protocols/test_default.py

```diff
@@ -104,3 +108,76 @@ def test_check_consistency_raises_exception_when_type_not_recognized():
     # mock the __eq__ method such that any kind of matching is going to fail.
     with mock.patch.object(DefaultMessage.Performative, "__eq__", return_value=False):
         assert not message._is_consistent()
+
+
+def test_default_dialogue():
```

Comment:
A lot of the tests will be repeated across protocols; so we should have some default test battery we can apply to each one of them

Reply:
yes, good idea. We could use `pytest.mark.parametrize